### ISSUE TITLE
Card info doesn't contain number of copies in the system gateway decks

### ISSUE THREAD

**Issue by qwitwa** [NetrunnerDB/netrunnerdb-api-server#328]
i.e. the dots and pluses at the bottom of the cards
![image](https://github.com/user-attachments/assets/80f8d58f-e3cd-427d-9366-9c0cc0ea4b34)

**Comment by qwitwa** [comment]
@plural do you think this would be better added in the json data itself for each card, or at a higher layer than that?

**Comment by plural** [comment]
I don't think this belongs on the card attributes.  It only applies to a tiny number of cards.

This could belong in a more explicit pre-built decklist object or somewhere else, though.
